kedro-datasets/kedro_datasets/spark/spark_stream_dataset.py

```diff
@@ -0,0 +1,158 @@
+"""SparkStreamingDataSet to load and save a PySpark Streaming DataFrame."""
+import os
+from typing import Any, Dict
+from copy import deepcopy
+import yaml
+from kedro.io import AbstractDataSet
+from pyspark import SparkConf
+from pathlib import PurePosixPath
+from pyspark.sql import SparkSession, DataFrame
+from yaml.loader import SafeLoader
+from kedro_datasets.spark.spark_dataset import _split_filepath
+
+class SparkStreamingDataSet(AbstractDataSet):
```

Comment:
Since the name is `SparkStreamingDataSet`, it's better to name the module `spark_streaming_dataset` instead of `spark_stream_dataset`, since this is how you named it in the docstring already.